### ISSUE TITLE
Exercise 7, Step 1, Task 9

### ISSUE THREAD

**Issue by ewittrock-uwm** [microsoft/MCW-Azure-Synapse-Analytics-and-AI#50]
The SQL Query returns this message:  Timeout: The SQL query could not be performed within 30 seconds. The dataset service has a timeout limit of 30 seconds which is not affected by the query timeout value supplied. It is advised to skip validation when registering datasets for such cases. Once the dataset is registered the custom timeout value will become effective.

The same Query seems to run forever if you try it in the Synapse Workspace itself.  Is there something wrong with the query?  Is there an alternative method to populate the data?

**Comment by codingbandit** [comment]
I am working through the lab to get to this point, I will see if I can recreate the issue.

**Comment by anushabc** [comment]
I am also getting the same issue in Exercise 7, Task 1, Step 9 

![exercise 7 issue](https://user-images.githubusercontent.com/66515706/98782488-14b5a700-241e-11eb-8a5a-73a68441ccc1.png)

**Comment by codingbandit** [comment]
I'm working on it, thank you for the confirmation :)

**Comment by codingbandit** [comment]
I will be updating this step in the next PR to the following:

9. In the next **Datastore selection** form, enter the following **SQL query**. Then expand the **Advanced settings** and enter **100** for the **Query timeout (seconds)** value. Select **Next**:

    ```sql
    SELECT  P.ProductId,P.Seasonality,S.TransactionDateId,COUNT(*) as TransactionItemsCount
    FROM wwi_mcw.SaleSmall S
    JOIN wwi_mcw.Product P ON S.ProductId = P.ProductId
    GROUP BY P.ProductId ,P.Seasonality,S.TransactionDateId
    ```
![image](https://user-images.githubusercontent.com/3979478/98870394-3d9b6380-2441-11eb-8665-e767e6855beb.png)

**Comment by DawnmarieDesJardins** [comment]
Closing as resolved with merge of PR #53
